From 13562141fe119e92c2dfb13dd2792e7072ce21ec Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 17 Apr 2017 14:52:02 +0800
Subject: [PATCH 1704/5242] MLK-14703-1 video: adv7535: add 'dsi-traffic-mode'
 new property

commit  a4ebffa61ecacc55985ef6825aff4c1775111031 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the new property 'dsi-traffic-mode' which is used to
set one of the three video data transmit modes:
   . Non-burst mode with sync pulse width
   . Non-burst mode with sync event
   . Burst mode

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 19eb160cfa365af6a377798cf44dc800f21ef51d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 046c3c2..4e7cf92 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -421,6 +421,7 @@
 			           * Only support CEA modes.
 			           * Reference mxc_edid.c
 			           */
+		dsi-traffic-mode = <0>;
 		bpp = <24>;
 		status = "disabled";
 	};
-- 
1.7.9.5

